<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>607</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>607</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>中心平台参数配置</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPServer">
     <attribute name="title">
      <string>平台中心服务器</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>网络</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TCP侦听端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerTcpPort"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>TCP待处理连接数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerTcpPendingConnections"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>51</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>UDP端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerUdpPort"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>UDP广播端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerUdpBroadcastPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>80</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>UDP组播IP  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerUdpMulticastIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>UDP组播端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerUdpMulticastPort"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>线程</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>通信线程管理连接数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerPeerThreadSocketCount"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>通信线程栈大小(K)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerPeerThreadStackSize"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>327</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>检测通信线程是否使用时间间隔(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerPeerThreadReleaseInterval"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>581</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>数据库</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>21</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>服务软件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbPServerDbType"/>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbUser"/>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>口令</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbPwd"/>
        </item>
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbName"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>80</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>线程池最大线程数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerThreadpoolMaxThreads"/>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>线程池程数超时(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerThreadpoolThreadsExpire"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>数据库对象释放间隔(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbObjReleaseInterval"/>
        </item>
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>数据库对象生命周期(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPServerDbObjLifeTime"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPDataReceiver">
     <attribute name="title">
      <string>平台数据接收服务器</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>线程</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>通信线程管理连接数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerPeerThreadSocketCount"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>通信线程栈大小(K)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerPeerThreadStackSize"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>327</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>检测通信线程是否使用时间间隔(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerPeerThreadReleaseInterval"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>网络</string>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>TCP侦听端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerTcpPort"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>TCP待处理连接数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerTcpPendingConnections"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>51</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>UDP端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerUdpPort"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>UDP广播端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerUdpBroadcastPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>80</y>
         <width>442</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>UDP组播IP  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerUdpMulticastIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>UDP组播端口    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerUdpMulticastPort"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>581</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>数据库</string>
      </property>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>21</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>服务软件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbPDataServerDbType"/>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbUser"/>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>口令</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbPwd"/>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbName"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>80</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>线程池最大线程数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerThreadpoolMaxThreads"/>
        </item>
        <item>
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>线程池程数超时(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerThreadpoolThreadsExpire"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>451</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>数据库对象释放间隔(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbObjReleaseInterval"/>
        </item>
        <item>
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>数据库对象生命周期(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPDataServerDbObjLifeTime"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPClient">
     <attribute name="title">
      <string>平台客户端</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>网络</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>UDP广播端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientUdpBroadcastPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>21</y>
         <width>430</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>TCP服务器IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientTcpIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>TCP服务器端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientTcpPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>52</y>
         <width>430</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>UDP服务器IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientUdpIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>UDP服务器端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientUdpPort"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>82</y>
         <width>430</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>UDP组播IP  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientUdpMulticastIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>UDP组播端口  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPClientUdpMulticastPort"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
